packages/xod-client/src/user/actions.js

```diff
@@ -0,0 +1,16 @@
+import url from 'url';
+import { UPDATE_COMPILE_LIMIT } from './actionTypes';
+import { XOD_COMPILE_DOMAIN } from './constants';
+
+export const updateCompileLimit = () => (dispatch) => {
+  fetch(url.format({
+    ...url.parse(process.env.XOD_COMPILE_DOMAIN || XOD_COMPILE_DOMAIN),
+    pathname: '/limits',
+  }))
+    .then(res => (res.ok ? res.json() : null))
+    .then(limit => dispatch({
+      type: UPDATE_COMPILE_LIMIT,
+      payload: limit,
+    }));
+};
+export default {};
```

Comment:
Do not export default nothing.

Reply:
But eslint throws errors when using not default for a single export. In the future, it will be easier to add new exports and remove empty default than to change the imports everywhere. These files will grow soon (login, account details and etc).

@nkrkv what do you think?

Reply:
Or maybe use `eslint-disable`?

Reply:
Ah, OK, a pattern. No problem.